telemetry/specs/INT.mdk

```diff
@@ -195,8 +193,13 @@ physical or logical port.
     The semantics of port identifiers may differ across devices, each INT hop
     chooses the port type it reports at each of the two levels.
 
+* Egress timestamp
+  - The device local time when the INT packet was processed by the egress
+physical or logical port.
```

Comment:
I like this definition better than the one in Appendix A.3. Can we replace the definition in Appendix A.3 with this one?

Reply:
Done.